worker-message-prioritization-target-capacity-calculators/src/main/java/com/github/workerframework/workermessageprioritization/targetqueue/TunedTargetQueueLengthProvider.java

```diff
@@ -75,7 +75,11 @@ public long getTunedTargetQueueLength(final String targetQueueName, final Target
     private double calculateCurrentTheoreticalConsumptionRate(final double currentConsumptionRate, 
                                                               final double currentInstances,
                                                               final double maxInstances){
-        return (currentConsumptionRate / currentInstances) * maxInstances;
+        if(currentInstances == 0D){
+            return currentConsumptionRate * maxInstances;
```

Comment:
```suggestion
            return currentConsumptionRate;
```

Is it better to underestimate consumption rate?
With your proposal if we were already running 4 instances then we'd over estimate the theoretical rate by a factor of 4.

I don't think either is perfect, and perhaps we need a new ticket to implement a new way of doing this calculation when the worker in question is not running in k8s.

Reply:
@andyreidz As discussed this morning we are going to go with overestimating the worker rather than starving the worker, potentially leaving the worker idle.